src/applications/check-in/reducers/index.js

```diff
@@ -75,8 +75,8 @@ const handler = Object.freeze({
   [TOKEN_WAS_VALIDATED]: tokenWasValidatedHandler,
   [TRIGGER_REFRESH]: triggerRefreshHandler,
   [SEE_STAFF_MESSAGE_UPDATED]: seeStaffMessageUpdatedHandler,
-  [RECEIVED_DEMOGRAPHICS_STATUS]: receivedDemographicsStatusHandler,
-  [UPDATE_FORM]: updateFormHandler,
+  [UPDATE_PRE_CHECK_IN_FORM]: updateFormHandler,
```

Comment:
Just to say it out loud, I think these could probably be combined(?) But that can be on the refactor selectors ticket, not this ticket.

Reply:
The thing that I couldn't think around was where to handle the different base URLs for the applications. I think once we get closer the unity of the apps there can be a dynamic URLS builder that could solve that problem and allow them to be combined.